src/core/scene/a-scene.js

```diff
@@ -169,7 +169,7 @@ var AScene = module.exports = registerElement('a-scene', {
         if (!camera || !canvas) { return; }
 
         // Update canvas.
-        if (!isMobile) {
+        if (!isMobile && !canvas.provided) {
```

Comment:
this assumes that those providing their own canvas are handling resizing themselves - which is probably a fair assumption